rdr_service/offline/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
```

Comment:
Why in put the file in 'rdr_service/offline'?

Reply:
Oops, meant to put it at the root. Moving it now.

Reply:
Ok, should be good now.